level2/modules/kernel/krn.asm

```diff
@@ -203,28 +207,39 @@ l@                  std       ,x++                store the vector value in the
                     bra       l@                  and do again
 done@               ldx       #$0000              start clearing at this address
                     ldy       #$2000              for this many bytes
+                    clra                          set D
```

Comment:
Why are you duplicating code? Lines 209-216 are the same as 232-238. There must be a better way to arrange the conditionals so that code isn't duplicated.

Reply:
Only in source, not the binaries but I can try something else.  Will require adding an "ifeq H6309" instead of what I did.  Meanwhile I am fighting with asmprettyprint.  It seems to be moving all the comments around when they used to be lined up.

Reply:
Per your request I used IFEQ H6309 to fix the problem instead of my previous method.  This gets rid of duplicated source.